explorer/src/main/java/com/scalar/client/tool/explorer/command/Validate.java

```diff
@@ -51,10 +52,18 @@
   public void run() {
     ExplorerExecutor executor =
         explorer -> {
+          StatusCode statusCode;
           try {
             for (String assetId : assetIds) {
-              explorer.validate(assetId);
-              System.out.println(assetId + " is not tampered");
+              statusCode = explorer.validate(assetId).getCode();
+
+              if(statusCode.equals(StatusCode.OK)) {
+                  System.out.println(assetId + " is not tampered");
+              } else if (statusCode.equals(StatusCode.ASSET_NOT_FOUND)) {
```

Comment:
I think we can simply do like this
```suggestion
              } else {
                System.out.println("validation failed: "  + StatusCode);
              }
```

Reply:
Thank you for review comments.
Updated review changes in fafec7963f9bf50fc4d58d6e8c3c956b9ef0458b